tests/codebuild/run_integtest.sh

```diff
@@ -27,7 +28,19 @@ function cleanup_default_namespace {
     set +e
     get_manager_logs
     delete_all_resources "default"
-    kustomize build "$path_to_installer/config/default" | kubectl delete -f -
+    generate_operator_installer_for_given_role "${default_operator_namespace}" "config/installers/rolebasedcreds" "${default_role_name}"
+    kubectl delete -f temp_file.yaml
+    rm temp_file.yaml
+}
+
+# A function that cleans up Jobs, CRDs and Operator in the default namespace
+# Parameter:
+#    $1: Name of the Role to be deleted
+function delete_generated_role {
+    local role_to_delete="${1}"
+    # Delete the role associated with the cluster thats being deleted
+    aws iam detach-role-policy --role-name "${role_to_delete}" --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
```

Comment:
Duplication of role policies, here. Perhaps we could store the list of policies in an array, then when we create we iterate over and apply - likewise for delete.

Reply:
Not sure i understand, the same policy is getting attached to each role.

Reply:
I foresee us adding extra permissions in the future (for CloudWatch logs and EC2 metrics). I suppose this is a nit since this isn't the case at the moment.